src/ScriptCs.Hosting/FileConsole.cs

```diff
@@ -35,9 +35,9 @@ public void WriteLine(string value)
             this.AppendLine(value);
         }
 
-        public string ReadLine()
+        public string ReadLine(string prompt)
```

Comment:
Correct, it is part of the general interface
On Mon, Nov 9, 2015 at 7:51 AM Kristian Hellang <notifications@github.com>
wrote:

> In src/ScriptCs.Hosting/FileConsole.cs
> <https://github.com/scriptcs/scriptcs/pull/1119#discussion_r44290238>:
>
> > @@ -35,9 +35,9 @@ public void WriteLine(string value)
> >              this.AppendLine(value);
> >          }
> >
> > -        public string ReadLine()
> > +        public string ReadLine(string prompt)
>
> It's just not used in the FileConsole.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scriptcs/scriptcs/pull/1119/files#r44290238>.
>